Single threaded build fails on Cetus

If the number of threads for all the components is set to 1 (by default it is set to 16) the ACME bulid fails on Cetus (with error messages like "undefined reference to `_xlsmpParSelf'") due to mixing of threaded (openmp) and non-threaded compiler flags.
This issue was noticed (thanks @bitshtgautam ) when manually modifying env_mach_pes.xml for one of the developer tests by setting the number of threads for all components to 1.

--- a/scripts/ccsm_utils/Machines/Depends.cetus
+++ b/scripts/ccsm_utils/Machines/Depends.cetus
@@ -6,9 +6,9 @@ shr_reprosum_mod.o: shr_reprosum_mod.F90
 	$(FC) -c $(INCLDIR) $(INCS) $(FFLAGS) $(FREEFLAGS) $(QSMPFLAGS) $<
 
 mo_sethet.o: mo_sethet.F90
-	$(FC) -c $(INCLDIR) $(INCS) $(FFLAGS) $(FREEFLAGS)  -qsmp=noauto:noomp $<
+	$(FC) -c $(INCLDIR) $(INCS) $(FFLAGS) $(FREEFLAGS) $(QSMPFLAGS) $<
 mo_drydep.o: mo_drydep.F90
-	$(FC) -c $(INCLDIR) $(INCS) $(FFLAGS) $(FREEFLAGS)  -qsmp=noauto:noomp $<
+	$(FC) -c $(INCLDIR) $(INCS) $(FFLAGS) $(FREEFLAGS) $(QSMPFLAGS) $<
 time_management.o: time_management.F90
-	$(FC) -c $(INCLDIR) $(INCS) $(FFLAGS) $(FREEFLAGS)  -qsmp=noauto:noomp $<
+	$(FC) -c $(INCLDIR) $(INCS) $(FFLAGS) $(FREEFLAGS) $(QSMPFLAGS) $<
 

